source/raw_assets/Persona-Pawel.user.js

```diff
@@ -0,0 +1,347 @@
+// ==UserScript==
+// @name         Persona Pawel
+// @namespace    https://github.com/alphagov/accessibility-personas
+// @version      1.0.0
+// @license      mixed (MIT, ISC, CC BY-SA)
+// @author       Crown Copyright (Government Digital Service), Metamatrix AB, Houshalter and benrwb
+// @description  Pawel, an autistic user - has the screen cluttered with various distractions to simulate finding it hard to concentrate, uses soothing colours to reduce potential stress
+// @homepageURL  https://alphagov.github.io/accessibility-personas/
+// @include      *
+// @grant        GM_addStyle
+// @nocompat     Chrome
+// ==/UserScript==
+
+/**
+ * Invert colours
+ * @author Crown Copyright (Government Digital Service)
+ * @license MIT
+ */
+
+GM_addStyle('html { filter: invert(1) hue-rotate(180deg) sepia(1) !important; } img, video, iframe, .wds-img-element.wds-img-animation-0::before { filter: invert(1) hue-rotate(180deg) !important; }');
+
+
+/**
+ * Beep (used within Distractions)
+ * @author Houshalter and benrwb [https://stackoverflow.com/a/29641185]
+ * @author Crown Copyright (Government Digital Service)
+ * @license CC BY-SA
+ */
+//if you have another AudioContext class use that one, as some browsers have a limit
+var audioCtx = new (window.AudioContext || window.webkitAudioContext || window.audioContext);
+
+//All arguments are optional:
+
+//start of the tone in milliseconds. Default is 0
+//duration of the tone in milliseconds. Default is 500
+//frequency of the tone in hertz. default is 440
+//volume of the tone. Default is 1, off is 0.
+//type of tone. Possible values are sine, square, sawtooth, triangle, and custom. Default is sine.
+//callback to use on end of tone
+function beep(start, duration, frequency, volume, type, callback) {
+  var oscillator = audioCtx.createOscillator();
+  var gainNode = audioCtx.createGain();
```

Comment:
```suggestion
  const gainNode = audioCtx.createGain();
```

Reply:
I've added it as part of the 'Add annoying beeps [...]' commit.